stock_orderpoint_purchase_link/models/stock_warehouse_orderpoint.py

```diff
@@ -13,3 +14,17 @@ class StockWarehouseOrderpoint(models.Model):
         copy=False,
         readonly=True,
     )
+    purchase_count = fields.Integer(compute="_compute_purchase_count")
+
+    @api.depends("purchase_line_ids")
+    def _compute_purchase_count(self):
+        for record in self:
+            record.purchase_count = len(record.purchase_line_ids.order_id)
+
+    def button_view_purchases(self):
+        self.ensure_one()
+        orders = self.purchase_line_ids.order_id
+        action = self.env.ref("purchase.purchase_rfq").sudo().read()[0]
```

Comment:
Please use _for_xml_id

Reply:
Well, in this case it's the same, isn't it?

Reply:
Odoo provides function to do that, so if the implementation changes, nothing to do here...

Reply:
OK, changed.